frontend/src/App.tsx

```diff
@@ -222,6 +222,9 @@ export class App extends PureComponent<Props, State> {
     ) {
       this.sendRerunBackMsg()
     }
+    if (this.props.s4aCommunication.currentState.forcedModalClose === true) {
```

Comment:
nit: (for better or for worse) it's slightly more canonical javascript to just write

```javascript
if (this.props.s4aCommunication.currentState.forcedModalClose)
```
here